README.md

```diff
@@ -8,31 +8,47 @@ This REDCap external module provides integration with Forte Research's OnCore. I
 - **SIP URL**: The URL of OnCore SIP (Study Information Portal), e.g. `https://oncore-test.ahc.ufl.edu/sip/SIPMain`. Returns **only** protocols open to enrollment
-- **OCR API URL**: The URL of OnCore API (Application Programming Interface), e.g. `https://oncore-test.ahc.ufl.edu/ocr/api/protocols`. Returns **all** protocols, requires API credentials
-  - **OCR API Username**: Your OnCore API user name
-  - **OCR API Key**: Your OnCore API key
+- **OCR API URL**: The URL of UF OCR OnCore API (Application Programming Interface), e.g. `https://oncore-test.ahc.ufl.edu/ocr/api/protocols`. Returns **all** protocols, requires UF OCR API credentials
+  - **OCR API Username**: Your UF OCR OnCore API user name
+  - **OCR API Key**: Your UF OCR OnCore API key
 - **Log requests**: Check this field to log all API requests (see Logs Page section) - this is useful for development purposes and testing
+- **Name of server variable used to populate Institution ID**: If you use an authentication mechanism that provides the institutional ID in a server environment variable, name that variable here to automatically populate a table of user names and Institution IDs.
 
 ![Config form](img/config_form.png)
 
-You will also need to provide the institution ID (e.g. an 8 digit UFID) for each user for them to access the protocol data. This is available at **Control Center > Enter User Institution IDs**. Users will _only_ be allowed to access protocol subject data if OnCore lists them as currently active staff on a protocol.
+### Institutional IDs
+
+The REDCap OnCore Client displays enrollment data only to REDCap users who are on OnCore's list of protocol staff. To do this, it reads the protocol staff list from OnCore and compares the institional ID of the logged-in REDCap user against the institional ID field in the OnCore study staff data. If there is a match, it displays the enrollees, otherwise it hides them. Note that REDCap super-users can always see the list of enrollees. 
+
+To do the comparison, REDCap needs to know the institional ID for the logged-in user. If you use an authentication mechanism that provides the institutional ID in a server environment variable, the OnCore Client can automatically write that value to the lookup tables it uses to verify access. Sites that use Shibboleth or LDAP for authentication are likely to have this functionality, but its implementation is site-specific. Whether your site provides an institutional for successfully-authenticated staff and what variable would hold that value are questions for your site's system staff. 
+
+If the institutional ID is _not_ available in an environment variable, you will need to provide that ID for each user for them to access the protocol data. This is available at **Control Center > Enter User Institution IDs**. Users will _only_ be allowed to access protocol subject data if OnCore lists them as currently active staff on a protocol.
```

Comment:
New functionality that was added yesterday.

```suggestion
If the institutional ID is _not_ available in an environment variable, you will need to provide that ID for each user for them to access the protocol data. This is available at **Control Center > Enter User Institution IDs**. Users will _only_ be allowed to access protocol subject data if OnCore lists them as currently active staff on a protocol or if they have Super User privileges.
```

On a related note, if using an environment variable, if data was entered in manually it will be overwritten. Should there be a check implemented? It may necessitate another row in the `oncore_staff_identifier` table so that manually entered information will not be overwritten, but automatically populated information will be refreshed when appropriate.